fern/api/openapi/openapi.json

```diff
@@ -1843,6 +2107,21 @@
           }
         }
       },
+      "WebhookPage": {
+        "title": "WebhookPage",
+        "required": ["items", "page", "size", "has_more"],
+        "type": "object",
+        "properties": {
+          "items": {
+            "title": "Items",
+            "type": "array",
+            "items": { "$ref": "#/components/schemas/Webhook" }
```

Comment:
@ajar98 any special tricks you did to get the array item types to propagate?

Reply:
ended up making a new pydantic model per API object for each paginated type - wasn't much extra code and worth it to not have to update the openapi spec manually